setup.cfg

```diff
@@ -31,6 +31,7 @@ zip_safe = False
 python_requires = >=3.6
 install_requires =
     Babel
+    beautifulsoup4
```

Comment:
Is it at all possible to achieve the necessary amount of HTML parsing without introducing new requirements?